.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cbits/tracy"]
+	path = cbits/tracy
+	url = https://github.com/wolfpld/tracy.git
```

Comment:
Would it be possible to simply vendor these files, or to search for them in `/usr/include` (so you'd be able to use the system-installed version of tracy)? Requiring the use of git submodules makes using Accelerate through either a regular `git clone` (e.g. through `extra-deps` in a `stack.yml` file or through a `repository` stanza in a `cabal..project` file) or a `git archive`/GitHub source tarball much more inconvenient since the submodule won't be included and the package thus won't build anymore without manually cloning the repo.

Reply:
Indeed! But the Setup.hs file will download it if it's not already present (so we need to remember to keep those two things in sync, which is a bit unfortunate).